### DISCUSSION TITLE
Event Gates should be able check the On/Off state of Glow Tiles and Indicator Lights as a form of memory storage

### DISCUSSION THREAD

**Discussion by Janssendel** [tsunamayo/Starship-EVO#5254] [Suggestions]
The Glow Tiles and Indicator Lights do seem rather useless in the current game due to the fact they can only show you if something is on or off, and can't be the trigger for event gates. My suggestion is that we allow Glow Tiles and Indicator Lights to trigger event gates by having them act as a form of physical memory storage via their on/off state. This will hopefully solve many gripes that players might have when building sophisticated mechanisms and add more functionality to them.

**Comment by ProPeach** [comment]
Can't you use whatever signal is turning the glow tile on/off instead? I'm not really sure what you mean - Any change in a glow tiles state would be caused by a logic pulse, so extracting that state into another pulse using an event gate doesn't seem useful when you could just use that original pulse.
The Event Gate is used for generating logic pulses from actions that we couldn't get pulses from before rather than this. Could you explain a use case in more detail?

I use glow tiles all the time for decorative purposes, they're fantastically useful
